src/app/app.component.ts

```
@@ -12,15 +22,76 @@
   constructor(
     private eventsService: EventsService,
     private messageService: MessageService,
-    private primengConfig: PrimeNGConfig
+    private primengConfig: PrimeNGConfig,
+    private siAuthService: SiAuthService,
+    private qboAuthService: QboAuthService,
+    private xeroAuthService: XeroAuthService,
+    private nsAuthService: NetsuiteAuthService,
+    private storageService: StorageService,
+    private router: Router,
+    private integrationsService: IntegrationsService,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)